exporter/datadogexporter/translate_traces_test.go

```diff
@@ -284,7 +285,11 @@ func TestTracesTranslationErrorsAndResource(t *testing.T) {
 
 	// ensure that env gives resource deployment.environment priority
 	assert.Equal(t, "test-env", datadogPayload.Env)
-	assert.Equal(t, 12, len(datadogPayload.Traces[0].Spans[0].Meta))
+
+	// ensure that env gives resource deployment.environment priority
+	assert.Equal(t, "test-version", datadogPayload.Traces[0].Spans[0].Meta["version"])
```

Comment:
We should pass a non-empty `Version` field to the config to actually test the priority

Reply:
👍  updated